example-get-started-experiments/code/notebooks/TrainSegModel.ipynb

```diff
@@ -237,19 +218,22 @@
     "            **fine_tune_args,\n",
     "            cbs=[DVCLiveCallback(live=live)])\n",
     "\n",
-    "        learn.export(fname=(models_dir / \"model.pkl\").absolute())\n",
-    "\n",
-    "        # add additional post-training summary metrics\n",
-    "        live.summary[\"evaluate/dice_multi\"] = evaluate(learn)\n",
-    "\n",
     "        # save model artifact to dvc\n",
+    "        models_dir = Path(\"models\")\n",
+    "        models_dir.mkdir(exist_ok=True)\n",
+    "        learn.export(fname=(models_dir / \"model.pkl\").absolute())\n",
+    "        torch.save(learn.model, (models_dir / \"model.pth\").absolute())\n",
     "        live.log_artifact(\n",
     "            str(models_dir / \"model.pkl\"),\n",
     "            type=\"model\",\n",
     "            name=\"pool-segmentation\",\n",
     "            desc=\"This is a Computer Vision (CV) model that's segmenting out swimming pools from satellite images.\",\n",
     "            labels=[\"cv\", \"segmentation\", \"satellite-images\", \"unet\"],\n",
-    "        )\n"
+    "        )\n",
+    "\n",
+    "    # add additional post-training summary metrics\n",
+    "    with Live(\"results/evaluate\") as live:\n",
+    "        live.summary[\"dice_multi\"] = evaluate(learn)"
```

Comment:
Experiment still gets saved here. `dvc_exp_save=False` above is only needed to avoid having two experiments saved for each run since there are two different live instances for train and eval.